atm.js

```diff
@@ -1,21 +1,86 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var atm = {
+sumChecking: 0,
+sumSavings: 0,
+listen:function() {
 
-}
+      $( "#checking input[value=Deposit]" ).on( "click" , atm.functions.DepositC );
+      $( "#checking input[value=Withdraw]" ).on("click" , atm.functions.WithdrawC );
+      $( "#savings input[value=Deposit]" ).on( "click" , atm.functions.DepositS );
+      $( "#savings input[value=Withdraw]" ).on("click" , atm.functions.WithdrawS );
 
-function checkingWithdrawal(amount){
+},
 
-}
+functions: {
+  DepositC: function() {
```

Comment:
JavaScript uses Capital function names to indicate contructor functions. Prefer lower case unless creating the object with the `new` keyword.